tests/test_api_log_injection.py

```
@@ -0,0 +1,60 @@
+            try:
+                client._make_request(endpoint=malicious_endpoint)
+            except Exception as e:
+                # We don't expect an exception here, but if one occurs, catch it
+                pass
+
+            # Check debug logs
+            found_ansi_in_log = False
+            for call in self.logger_mock.debug.call_args_list:
+                args, _ = call
+                log_msg = args[0]
+                print(f"Log: {repr(log_msg)}")
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)